api/db-cdr/generate-cdr/bq-schemas/concept.json

```diff
@@ -33,21 +33,6 @@
 	"mode": "REQUIRED", 
 	"name": "concept_code", 
 	"type": "STRING"
-	}, 
-	{
-	"mode": "REQUIRED", 
```

Comment:
Why is this being removed? 

These columns are a part of the standard OMOP schema; our code expects them to be there.

While I wouldn't do it in this PR, we should probably change our scripts to use the schema files here:

https://github.com/all-of-us/curation/blob/develop/data_steward/resources/fields

which do include these columns.

Reply:
We decided to remove them from the cloudsql tables a while back because they aren't used by workbench or data browser and it makes the cloudsql tables much smaller .  If i recall me and you brian all agreed it was ok to remove. We removed them from cloudsql a while back but forgot to pull them from the generated bq tables that we make our cloudsql from. 

Make sense ? So this is a bug fix ...

Reply:
Removing them from Cloud SQL is different than removing them from BigQuery. And for at least BigQuery, if you don't use these columns in your query, they shouldn't impact the performance of queries on these tables. So I would not drop them here.

Reply:
They still exist in the cdr .. Just not in the counts dbs we generate.  This is just in the intermediary bq tables. I thought we wanted those to be exactly like our cloudsql. And the import export relies on that fact. So if we put them back in we will have to tweak the import export csv code.   I see it as not something worth doing right now but if we need them in later add them back.  You're call though Dan. I don't mind either way. Unless its really hard to pick and choose what you export to a csv from a bq table :)

Reply:
OK... I thought this was somehow being used as a substitute for the CDR itself. If it's just for intermediate tables, no objections here -- please disregard.

Reply:
Cool. Nope. I would like to adopt the term count_cdrs or count_dbs to avoid this confusion. Or public_cdr .... It's just the data from the cdr available to public. We will probably allow them to download the csvs and mysqldump from the same public bucket for a cdr.

Reply:
Okay so I changed the schema for concept and concept_relationship to remove valide_start_date, valid_end_date and invalid_reason as the data import was weird without this removal. New data generation is under process.